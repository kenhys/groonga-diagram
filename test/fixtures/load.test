load --table Site
[
{"_key":"http://example.org/", "title":"example org"}
]
