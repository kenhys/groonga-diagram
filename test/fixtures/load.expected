load --table Site
[
{"_key":"http://example.org/", "title":"example org"}
]
[[0,0.0,0.0],1]
